src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,22 +24,22 @@
 public class RobotContainer {
 
   //Controllers
-  private final Joystick m_joystick = new Joystick(CONTROLS.JOYSTICK_PORT);
-	private final Joystick arcade_joystick = new Joystick(CONTROLS.ARCADE_JOYSTICK_PORT);
+  private final Joystick movement_Joystick = new Joystick(CONTROLS.JOYSTICK_PORT);
```

Comment:
In Java, we generally prefer lowerCamelCase for naming of variables and methods, and UpperCamelCase for classes and interfaces.

Underscores are used sparingly, only for constants, which use a style like CONSTANT_NAME_HERE.